rfcs/20210510-sum-with-zero-point/README.md

```diff
@@ -0,0 +1,97 @@
+# Proposal for support sum with zero-point
+
+## Motivation
+The motivation to provide such support is to enable dequantize the 
+asymmetrically quantized sum's src1 tensor to f32 domain before performing the 
+sum operation. In general we want to support following operation:
+`dst[:] := scale * (dst[:] - zero_point) + op(...)`
+
+## Proposal
+The proposal is to extend API by adding the possibility of use such 
+functionality. Support for reference solutions should also be added. For 
+consistency also need to implement zero points support in standalone sum 
+primitive. The solution should also include the benchdnn extension.
+
+## Design
+
+Currently, the sum operation also supports scale. Zero point support should 
+be designed to keep backward compatibility.
+
+### API extension
+
+1. Overload `append_sum(scale, dt)` function by adding 
+`append_sum(scale, zp, dt)` instance for C++ API. Add 
+`dnnl_post_ops_append_sum_v3` for C API (preferred)
+    - Pros:
+        - It is backwards compatible.
+        - Allows to use zero point without scale (scale == 1.0).
+    - Cons:
+        - It is not the simplest solution
+
+2. Extend `append_sum(scale, dt)` function to 
+`append_sum(scale, zp, dt)` and `dnnl_post_ops_append_sum_v2` to support zero 
+point.
+    - Pros:
+        - Very simple solution.
+        - Universal solution.
+    - Cons:
+        - It is not backwards compatible.
+
+3. Add possibility to use `dnnl_primitive_attr_set_zero_points` with sum.
+    - Pros:
+        - It will allow to use mask and array of zero-point values (instead of 
+          one scalar value).
+    - Cons:
+        - More complicated solution.
+        - Probably requires backward incompatibility or adding a new function.
+        - Probably not needed in real topologies.
+
+### Sum zero-point design
```

Comment:
@igorsafo @mgouicem  At now I see other problem, standalone zero point use `arg` to select parameter. Here we want to use zero_point on src (previous dst) but I think this cannot be assumed. I added these considerations to the rfc in `Zero-point design` part.

Reply:
I am inclined to this option too.